shavar/__init__.py

```diff
@@ -53,12 +54,23 @@ def get_configurator(global_config, **settings):
     return config
 
 
+def group_errors(event, hint):
+    if 'logentry' in event and 'message' in event['logentry']:
+        message = event['logentry']['message']
+        pattern = 'Invalid RANGE "[0-9]{10}'
```

Comment:
I reflexively use `r` for patterns, and you get better syntax highlighting. I think it doesn't matter in this case because there are no slashes, however.
```suggestion
        pattern = r'Invalid RANGE "[0-9]{10}'
```

Reply:
Ended up not needing `group_errors` since `capture_exception` properly groups the events. Thanks for bringing it up! I'll remember to do this next time I use regex patterns.